crates/pink-drivers/system/lib.rs

```diff
@@ -209,6 +209,12 @@ mod system {
             pink::upgrade_runtime(version);
             Ok(())
         }
+
+        /// Check if the code is already uploaded to the cluster with given code hash.
+        #[ink(message)]
+        fn code_exists(&self, code_hash: [u8; 32], code_type: CodeType) -> bool {
+            pink::ext().code_exists(code_hash.into(), code_type.is_sidevm())
```

Comment:
Where is it implemented?

Reply:
The underlying implementation is in PR #1168